client/components/forms/EmployeeCertificationForm.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+
+import { Button } from "baseui/button";
+import { FlexGrid, FlexGridItem } from "baseui/flex-grid";
+import { Heading, HeadingLevel } from "baseui/heading";
+import { Form, Formik, FormikConfig } from "formik";
+
+import { Loading } from "../common/Loading";
+import { useStyletron } from "baseui";
+import { ComboboxField } from "../fields/ComboBoxField";
+import {
+  EmployeeCertificationInput,
+  useCertificateProviderQuery,
+  useCertificationsProvidersQuery,
+} from "../../graphql/types";
+import { DatePickerStrField } from "../fields/DatePickerStrField";
+
+interface EmployeeCertificationFormProps
+  extends FormikConfig<EmployeeCertificationInput> {
+  action: string;
+  initialCertificataionProvider?: number;
+}
+
+export const EmployeeCertificationForm: React.FC<EmployeeCertificationFormProps> = ({
+  action,
+  initialCertificataionProvider,
+  ...props
+}) => {
+  const [css, theme] = useStyletron();
+
+  // Certificate Provider
+  const [
+    certificationProviderId,
+    setCertificationProviderNameValue,
```

Comment:
certifcationProvider
setCerticationProvider

or 

certifcationProviderId
setCerticationProviderId

based on what you named the initial property and its type.